src/providers/mbed_provider/mod.rs

```diff
@@ -56,13 +57,17 @@ const SUPPORTED_OPCODES: [Opcode; 7] = [
     Opcode::ListOpcodes,
 ];
 
+const KEY_SLOT_COUNT: isize = 32;
```

Comment:
Is there a way to read that value from the FFI?

Reply:
or maybe put it in the constants file instead?

Reply:
Not through FFI, as it is not directly exposed in the header files, but I added it in the constants file.